packages/react-components/react-table/src/components/DataGridHeaderCell/DataGridHeaderCell.types.ts

```diff
@@ -9,7 +9,7 @@ export type DataGridHeaderCellSlots = TableHeaderCellSlots;
 /**
  * DataGridHeaderCell Props
  */
-export type DataGridHeaderCellProps = TableHeaderCellProps;
+export type DataGridHeaderCellProps = Omit<TableHeaderCellProps, 'sortable'>;
```

Comment:
This will expand `TableHeaderCellProps`, it might be a problem if the root is an union (which is the case here, as root is `Slot<'th', 'div'>` ). Would be nice to verify

Reply:
Seems like `HTMLTableHeaderCellElement` is equivalent to `HTMLElement`. I don't think this will explode anywhere